features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -90,7 +90,10 @@ export function SidebarNavigation() {
               iconSrc="/icons/arrow-left.svg"
               isCollapsed={isSidebarCollapsed}
               onClick={() => toggleSidebar()}
-              className={styles.collapseMenuItem}
+              className={classNames(
+                styles.collapseMenuItem,
+                isSidebarCollapsed && styles.iconRotate,
+              )}
```

Comment:
Added a conditional render based on the boolean `isSidebarCollapsed` so that the css transform would apply to the `iconSrc` when `true`.